projection/validator_delegation/migrations/20220117104331_create_view_vd_delegations.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE EXTENSION btree_gist;
+
+CREATE TABLE view_vd_delegations (
```

Comment:
suggest to change it in completed form instead of `vd`

Reply:
Updated. And the similar places below